### PR TITLE
LPS-192286 Replace Publish button by a dropdown

### PR DESCRIPTION
**Pull request by sofiamendozalr**
https://liferay.atlassian.net/browse/LPS-192286
- LPS-192286 Change publish button for a dropdown menu
- LPS-192286 Add getEditKBArticleActionDropdownItems method
- LPS-192286 Force submit behavior in publishButtonOnClick
- LPS-192286 Add FF

image

### CODE REVIEW

**Comment by liferay-lima** [comment]
ci:test:sf

**Comment by liferay-lima** [comment]
ci:test:relevant

File: modules/apps/knowledge-base/knowledge-base-web/src/main/resources/META-INF/resources/admin/common/edit_kb_article.jsp [hunk 1]
```diff
@@ -56,16 +56,33 @@ if (editKBArticleDisplayContext.isPortletTitleBasedNavigation()) {
...
-							displayType="primary"
-							id='<%= liferayPortletResponse.getNamespace() + "publishButton" %>'
-							label="<%= editKBArticleDisplayContext.getPublishButtonLabel() %>"
-							name="publishButton"
-							small="<%= true %>"
-							type="submit"
-						/>
+						<c:choose
+						>
+							<c:when test='<%= FeatureFlagManagerUtil.isEnabled("LPS-192286") %>'>
```
Comment (AliciaGarciaGarcia) [inline modules/apps/knowledge-base/knowledge-base-web/src/main/resources/META-INF/resources/admin/common/edit_kb_article.jsp:61]: Hi @sofiamendozalr 

We usually use for the FF the LPS of the epic or the Main Story.
On this case I think that LPS-188060 is more accurate

**Review by AliciaGarciaGarcia — APPROVED** [review]
except for the FF number, LGTM

File: modules/apps/knowledge-base/knowledge-base-web/src/main/resources/META-INF/resources/admin/common/edit_kb_article.jsp (hunk 1 above)
Comment (ambrinchaudhary) [reply modules/apps/knowledge-base/knowledge-base-web/src/main/resources/META-INF/resources/admin/common/edit_kb_article.jsp:61]: that is true, FF should point to the epic number

**Comment by ambrinchaudhary** [comment]
see https://github.com/liferay-lima/liferay-portal/pull/4604

**Pull request closed without merging**